apps/e2e/ag-grid-storybook-e2e/src/integration/data-entry-grid.component.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('ag-grid-storybook', () => {
+  E2eVariations.forEachTheme((theme) => {
+    describe(`in ${theme} theme`, () => {
+      it('should edit date and lookup fields', () => {
+        cy.viewport(1300, 1300).visit(
+          `/iframe.html?globals=theme:${theme}&id=dataentrygridcomponent-dataentrygrid--data-entry-grid-date-and-lookup`
+        );
+        cy.waitForBlackbaudSans();
+        cy.get('#ready')
+          .should('exist')
+          .end()
+
+          .window()
+          .scrollTo('bottom')
+          .end()
+
+          // Activate a lookup field.
+          .get('#editLookup div[row-id="alexape01"] > div[col-id="name"]')
+          .should('be.visible')
+          .click()
+          .end()
+
+          .window()
+          .scrollTo('bottom')
+          .end()
+
+          // Activate a date field without the calendar.
+          .get('#editNumber div[row-id="aparilu01"] > div[col-id="birthday"]')
+          .should('be.visible')
+          .click()
+          .end()
+
+          .window()
+          .scrollTo('top')
+          .end()
+
+          // Activate a date field.
+          .get('#editDate div[row-id="berrayo01"] > div[col-id="birthday"]')
+          .should('be.visible')
+          .click()
+          .end()
+
+          // Open the calendar.
+          .get('#editDate .ag-popup-editor button[aria-label="Select date"]')
+          .should('be.visible')
+          .click()
+          .end()
+          .get('.ag-custom-component-popup .sky-datepicker-btn-selected')
+          .should('be.visible')
+          .should('contain.text', '12')
+          .end()
+
+          // Screenshot the three grids with active editors.
+          .get('sky-preview-wrapper')
+          .should('exist')
+          .should('be.visible')
+          .screenshot(
+            `dataentrygridcomponent-dataentrygrid--date-and-lookup-${theme}`,
+            {
+              overwrite: true,
+              disableTimersAndAnimations: true,
+              capture: 'runner',
+            }
+          )
+          .percySnapshot(
+            `data entry grid: edit date, number, and lookup fields -- ${theme}`,
+            {
+              widths: E2eVariations.DISPLAY_WIDTHS,
+            }
+          );
+      });
+
+      it('should edit lookup field, showing results', () => {
```

Comment:
This test shows a basic text input active as well as a lookup field with results.